services/movie.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        movies = movies.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    elif genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+    elif actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+    return movies

```

Comment:
You can simplify your code
```suggestion
    if genres_ids:
        movies = movies.filter(genres__id__in=genres_ids)
    if actors_ids:
        movies = movies.filter(actors__id__in=actors_ids)
    return movies

```